api/config/services_test.yaml

```diff
@@ -0,0 +1,9 @@
+services:
+    Psr\Log\NullLogger: ~
+
+    # Disable logger to avoid showing errors during tests
+    # see https://github.com/symfony/symfony/issues/28023
+    logger: '@Psr\Log\NullLogger'
+
+    # No cache for tests
+    App\Repository\TopBook\TopBookDataInterface: '@App\Repository\TopBook\TopBookDataRepository'
```

Comment:
Using cache on tests is not a bad thing, Symfony handles its ttl. Please remove this line

Reply:
It is a problem locally because that means you have to purge the cache before each test. Or running twice the tests could lead to different results.